great_expectations/rule_based_profiler/profiler.py

```diff
@@ -144,6 +149,12 @@ def profile(
             expectation_suite_name=expectation_suite_name
         )
 
+        if include_citation:
+            expectation_suite.add_citation(
+                comment="Add Profiler config in metadata",
```

Comment:
My understanding is that the comment is added to the Suite as a bit of metadata about the metadata of the rest of the citation. Is that correct? If so, something like this might work
```suggestion
                comment="Suite created by Rule-based profiler with the following config",
```

Reply:
How about we capitalize all three words in the name "Rule-Based Profiler"? :) Thanks.

Reply:
Made this change as well.